scripts/verification/collateral-plugins/verify_stargate_usdbc.ts

```diff
@@ -0,0 +1,114 @@
+import hre from 'hardhat'
```

Comment:
I think this file should be named verify_stargate_usdc as it works for both right? To match the one in the deployment script